ci/scripts/go-executable-build.sh

```
@@ -0,0 +1,39 @@
+if [[ -z "$package" ]]; then
+  echo "usage: $0 <package-name>"
+  exit 1
+fi
+package_name="$package"
+echo "cleaning dist..."
+# Use -rf to force removal without errors if directory doesn't exist
+rm -rf "$output_path"
+# Create directory with -p to create parent directories safely
+mkdir -p "$output_path"
+# Add error checking for directory creation
+if [ $? -ne 0 ]; then
```

Shellcheck (0.10.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)